manifests/g/GitHub/ClassroomAssistant/2.0.3/GitHub.ClassroomAssistant.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: GitHub.ClassroomAssistant
+PackageVersion: 2.0.3
+PackageLocale: en-US
+Publisher: GitHub
+PackageName: Classroom Assistant

```

```suggestion
PackageName: classroom-assistant
```
To match the Add and Remove Programs table.